frontend/www/js/omegaup/admin/support.ts

```diff
@@ -118,6 +126,40 @@ OmegaUp.on('ready', () => {
                 .catch(ui.apiError);
             }
           },
+          'search-contest': (contestAlias: string): void => {
+            adminSupport.contestFound = false;
+            adminSupport.contestTitle = null;
+            adminSupport.isContestRecommended = false;
+
+            api.Contest.details({ contest_alias: contestAlias })
+              .then((data) => {
+                adminSupport.contestAlias = contestAlias;
+                adminSupport.contestTitle = data.title;
+                adminSupport.contestFound = true;
+                adminSupport.isContestRecommended = !!data.recommended;
```

Comment:
Here, `data.recommended` is already being sent as a boolean, so there's no need to convert it
```suggestion
                adminSupport.isContestRecommended = data.recommended;
```